games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A 2D arcade shoot-em-up with 41 levels and more than 2000 sprites"
+DESCRIPTION="Made by TLK Games."
```

Comment:
That's hardly a description...
See the game's website and documentation at http://linux.tlk.fr/games/Powermanga/documentation/ to find interesting info about the backstory, gameplay and features.

Reply:
True... I've added a more extensive one now.